Apps/playitgg/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+name: playitgg
```

Comment:
This should be big-bear-playitgg. So it doesn't interfere with another app store.